src/applications/static-pages/facilities/vet-center/createExpandableOperatingStatus.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+export default async function createExpandableOperatingStatus() {
+  const statusWidgets = Array.from(
+    document.querySelectorAll(
+      'div[data-widget-type^=expandable-operating-status]',
+    ),
+  );
+
+  if (statusWidgets.length) {
+    const {
+      default: ExpandableOperatingStatus,
+    } = await import('../../shared/ExpandableOperatingStatus');
+
+    statusWidgets.forEach(el => {
+      if (
+        !el.attributes.status.nodeValue ||
+        el.attributes.status.nodeValue === 'normal'
```

Comment:
Always a good idea to use optional chaining when you have a long string of object references, to avoid null reference errors.

i.e. `el.attributes?.status?.nodeValue`.